libraries/ESP8266WiFi/src/WiFiClientSecureBearSSL-fp.cpp

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
Can we please just put these couple methods inside the main WiFiClientSecureBearSSL.cpp file?  I don't see a good reason for these to live outside that file.